content/en/docs/setup/install/istioctl/index.md

```diff
@@ -18,6 +18,10 @@ Using these instructions, you can select any one of Istio's built-in
 [configuration profiles](/docs/setup/additional-setup/config-profiles/)
 and then further customize the configuration for your specific needs.
 
+Full customization of the installation can be done through the
+[`IstioOperator` API](/docs/reference/config/istio.operator.v1alpha1/)
+(also see [protobuf format](https://github.com/istio/api/blob/release-1.5/operator/v1alpha1/operator.proto)).
```

Comment:
Why should users also be told to look at the proto? Is there stuff missing in the API doc?

Reply:
No but the API doc unhelpfully reorders the proto alphabetically which makes it much harder to understand.

Reply:
John recently sent a PR to change the doc generator to use the proto order, so maybe it's not needed if that's fixed? https://github.com/istio/tools/pull/808

Reply:
It's still sorted in 1.5 docs but I assume this is temporary until the PR takes effect - I've removed the proto link.

Reply:
Actually, this brings up a good question. Are istio/tools from the master branch used for release-1.5 docs, or does the tools PR need to be cherrypicked?
^^^ @howardjohn ?

Reply:
Its pinned per version. Lets get it end to end into the doc before we try to cherrypick though, to make sure it looks a reasonable way